ui/src/style/layout/sidebar.scss

```diff
@@ -310,3 +292,52 @@ span.icon.sidebar--icon.sidebar--icon__superadmin {
   .fancy-scroll--thumb-h {display: none !important;}
   .fancy-scroll--thumb-v { @include gradient-v($g20-white,$c-neutrino); }
 }
+
+.sidebar-menu--section__custom-links  { order: 0; }
+.sidebar-menu--item__link-name        { order: 1; }
+.sidebar-menu--section__switch-orgs   { order: 2; }
+.sidebar-menu--scrollbar              { order: 3; }
+.sidebar-menu--section__account       { order: 4; }
+.sidebar-menu--provider               { order: 5; }
+.sidebar-menu--item__logout           { order: 6; }
+.sidebar-menu--heading                { order: 7; }
+.sidebar-menu--triangle               { order: 8; }
+
+.sidebar-menu--section__custom-links:after {
+  display: none;
+  border-top-right-radius: $radius;
+}
+.sidebar-menu--user-nav {
+  top: initial;
+  bottom: 0;
+}
+
+@media only screen and (min-height: 750px) {
```

Comment:
As mentioned in the comment in the PR I'd up this to `800px` to allow for at most 2 custom links to not get cropped

Reply:
Fixed!